packages/widgetbook/lib/src/routing/widgetbook_shell.dart

```diff
@@ -63,3 +78,144 @@ class WidgetbookShell extends StatelessWidget {
     );
   }
 }
+
+class MobileWidgetbookShell extends StatefulWidget {
+  const MobileWidgetbookShell({
+    super.key,
+    required this.child,
+  });
+
+  final Widget child;
+
+  @override
+  State<MobileWidgetbookShell> createState() => MobileWidgetbookShellState();
+}
+
+class MobileWidgetbookShellState extends State<MobileWidgetbookShell> {
+  int _pointerCount = 0;
+  bool isBottomBarVisible = true;
+
+  void handlePointerDown(PointerDownEvent event) {
+    setState(() {
+      _pointerCount++;
+    });
+
+    if (_pointerCount == 4) {
+      setState(() {
+        isBottomBarVisible = !isBottomBarVisible;
+      });
+    }
+  }
+
+  void handlePointerUp(PointerUpEvent event) {
+    if (_pointerCount == 0) {
+      return;
+    }
+    setState(() {
+      _pointerCount--;
+    });
+
+    if (_pointerCount == 0) {
+      setState(() {
+        isBottomBarVisible = !isBottomBarVisible;
+      });
+
+      print('_pointerCount: $_pointerCount');
+
+      print('isBottomBarVisible: $isBottomBarVisible');
+    }
+  }
+
+  void _showModalBottomSheet(BuildContext context, Widget child) {
+    showModalBottomSheet<void>(
+      context: context,
+      builder: (context) {
+        return child;
+      },
+    );
```

Comment:
We can call `showModalBottomSheet` directly, since no special logic is done in `_showModalBottomSheet`.